plato/draw/vispy/Box.py

```diff
@@ -0,0 +1,11 @@
+from ... import draw
+from .internal import GLPrimitive, GLShapeDecorator
+from .Lines import Lines
+
+@GLShapeDecorator
+class Box(draw.Box, Lines):
+    __doc__ = draw.Box.__doc__
+
+    def __init__(self, *args, **kwargs):
+        GLPrimitive.__init__(self)
```

Comment:
@klarh We may want to change the inheritance in `vispy` to look more like `fresnel`, with the new structure. I'll leave that to you, if you agree with the design change -- I often have difficulty testing the vispy backend.